config.json

```diff
@@ -12,18 +12,13 @@
   "version": 3,
   "online_editor": {
     "indent_style": "space",
-    "indent_size": 0
+    "indent_size": 4,
+    "highlightjs_language": "roc"
   },
   "files": {
-    "solution": [
-      "%{pascal_slug}.roc"
-    ],
-    "test": [
-      "%{kebab_slug}-test.roc"
-    ],
-    "example": [
-      ".meta/Example.roc"
-    ]
+    "solution": ["%{pascal_slug}.roc"],
+    "test": ["%{kebab_slug}-test.roc"],
```

Comment:
Each time we run something like

```
bin/create-exercise
```

(or Erik runs some maintenance update on every track)

the `"files":  { ... }` part reverts to using multiple lines, and the `tags` get sorted alphabetically. 

I added commit `b97cc7c` to bring it to "canonical" form.